src/score-bot/components/report-item/report-item.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { useEffect } from "react";
+import * as semver from "semver";
+import { IReportItemInitInteractive,
+         addGetReportItemAnswerListener,
+         sendReportItemAnswer,
+         getClient,
+         IReportItemAnswerItem } from "@concord-consortium/lara-interactive-api";
+import { IAuthoredState, IInteractiveState } from "../types";
+
+interface Props {
+  initMessage: IReportItemInitInteractive;
+}
+
+export const ReportItemComponent: React.FC<Props> = (props) => {
+
+  useEffect(() => {
+    addGetReportItemAnswerListener<IInteractiveState, IAuthoredState>((request) => {
+      const {platformUserId, version} = request;
+
+      if (!version) {
+        // for hosts sending older, unversioned requests
+        // tslint:disable-next-line:no-console
+        console.error("ScoreBOT Report Item Interactive: Missing version in getReportItemAnswer request.");
+      }
+      else if (semver.satisfies(version, "2.x")) {
+        const items: IReportItemAnswerItem[] = [];
+        items.push({type: "answerText"});
```

Comment:
This API is a bit confusing, as when we send `{type: "answerText"}` item, the actual answer text is pulled on the report side from the interactive state. 

I think it'd be better if we sent `answerText` here, but it'd require changes to API. I might have to do it because of other upcoming work.